Rakefile

```diff
@@ -1 +1,84 @@
+#  Copyright 2015 Puppet Community
```

Comment:
In addition to a copyright holder needing some sort of legal existence, I believe that in most jurisdictions a copyright assignment or transfer requires an executed agreement.  As @daenney said, this gets messy fast.

A framework for enabling new maintainers (read: committers) for modules, similar to how jenkins operates, is a very good idea.  IANAL but I suspect that "puppet community" may be too generic to become a registered trademark in the US.

Specific to this PR, I'd suggest using your name (or your employers, if they own the work product).  My preference is not to copy the APL 2.0 boilerplate into every file, which seems to be a common non-practice in the puppet world, but I'm not sure how that jives with section `4b` -- which I very rarely see followed.